doc/man7/EVP_KDF_PBKDF2.pod

```diff
@@ -34,9 +34,22 @@ The supported controls are:
 
 These controls work as described in L<EVP_KDF_CTX(3)/CONTROLS>.
 
-B<iter> is the iteration count and its value should be greater than or equal to
-1. RFC 2898 suggests an iteration count of at least 1000.  The default value is
-2048.  Any B<iter> less than 1 is treated as a single iteration.
+=item B<EVP_KDF_CTRL_SET_PBKDF2_PKCS5_MODE>
+
+This control expects one argument: C<int mode>
+
+This control can be used to turn off SP800-132 compliance checks. This is needed
+in some cases, including but not limited to, PKCS5 related test vectors and
+scrypt.
+The compliance checks performed are:
+A minimum iteration count of 1000 and length of the salt being at least
+128 bits.
+
+The default value is 0. Use 1 to disable the checks.
```

Comment:
So, this sounds like having the default like this is a breaking change?

Reply:
enc uses the PKCS5_PBKDF2_HMAC() method which sets this control to be 1, as does scrypt.

Reply:
Added note to CHANGES